common/table.js

```diff
@@ -806,6 +806,36 @@
                 _callonSelectedRowsChanged(scope, [tuple], isSelected);
             };
 
+            // NOTE: For some reason this is being registered/triggered twice when the download widget is part of the record table (main body of record)
```

Comment:
This code should be somewhere else. I know we're including this file in all our apps but it's supposed to be for recordset and recordset table.